QUICKSTART.rst

```diff
@@ -136,10 +166,11 @@ What's next?
     changes since the last release              ``CHANGES.md``
 =============================================== =========================
 
-.. _platforms: http://chapel.cray.com/docs/1.13/platforms/index.html
-.. _chapel.cray.com/docs: http://chapel.cray.com/docs/1.13/
-.. _doc/building.rst: http://chapel.cray.com/docs/1.13/usingchapel/building.html
-.. _doc/compiling.rst: http://chapel.cray.com/docs/1.13/usingchapel/compiling.html
-.. _doc/executing.rst: http://chapel.cray.com/docs/1.13/usingchapel/executing.html
-.. _doc/debugging.rst: http://chapel.cray.com/docs/1.13/usingchapel/debugging.html
-.. _doc/bugs.rst: http://chapel.cray.com/docs/1.13/usingchapel/bugs.html
+.. _platforms: http://chapel.cray.com/docs/1.14/platforms/index.html
```

Comment:
> We could do that. I thought perhaps we wanted the QUICKSTART file to point to web pages for some reason

The reason was that this file used to be our README.rst which wouldn't have supported the cross-references. The links can still give the path to the files, e.g. 

```
See  :ref:`doc/prereqs.rst <chpl-prereqs>` for more details